### PR TITLE
[chart.js] Added scale property, used by radio chart

### PR DESCRIPTION
**Pull request by ElianCordoba**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <https://www.chartjs.org/docs/latest/charts/radar.html#scale-options>
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by ElianCordoba** [comment]
Screen Shot 2019-08-14 at 11 10 14
Screen Shot 2019-08-14 at 11 09 33
 
Result of using
```typescript
scale: {
  ticks: {
    display: false
  }
}
``` 

Chart generated with ng2-charts

File: types/chart.js/index.d.ts [hunk 1]
```diff
@@ -239,6 +240,7 @@ declare namespace Chart {
         animation?: ChartAnimationOptions;
         elements?: ChartElementsOptions;
         layout?: ChartLayoutOptions;
+        scale?: ChartScales;
```
Comment (lekoaf) [inline types/chart.js/index.d.ts]: `scale` and `scales`, which is correct? Can the either of them be removed?

Comment (ElianCordoba) [reply types/chart.js/index.d.ts]: Both are used, ´scale´ is used in the radio chart while `scales` is used, for example, in the bar chart.

Comment (adripanico) [reply types/chart.js/index.d.ts]: It is true, both ```scale``` and ```scales``` exist, but I think they don't implements the same interface. According to the documentation you provided, there is only one property:

```
options = {
    scale: {
        // Hides the scale
        display: false
    }
};
```

Comment (ElianCordoba) [reply types/chart.js/index.d.ts]: You are right, now I updated it.

**Pull request merged**